### ISSUE TITLE
The app no longer shows certain verses and it just shows white spaces

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#515]
Via email:

> I am a frequent user of AND Bible app and I have enjoyed using it till the last update. The app is broken after the last update. I have attached a screenshot of the bug herewith. The app no longer shows certain verses and it just shows white spaces (as shown in the screenshot). I tried wiping the app data/cache and reinstalling the app but it did not help. I am looking forward to receiving a fix. I hope my feedback was helpful and kudos to your great work. 

![Screenshot_2020-01-07-00-20-36](https://user-images.githubusercontent.com/5811789/71872917-6e768200-3126-11ea-9d4a-694424681539.jpg)

Moved from #510 
Might be related: #488
Reported for 333

**Comment by tuomas2** [comment]
If this is symptom of Chrome bug (as described in #488) we can't do anything. Let's see if we can reproduce this. Screen shape seem to be pretty weird in this particular case.

**Comment by bvahedy** [comment]
Yes this seems to be the chrome bug, it's the screen taking long to refresh, his device/os delay may be longer so he could get a screen shot. I can see what he sees but my screen fills up faster. For some reason turning off all the options like displaying the morph and one verse per line, etc. makes it slower.

**Comment by tuomas2** [comment]
What makes me think it might not be the chrome bug is that he was (supposedly) upgrading from 3.1.305 and noticed issue after upgrade.

**Comment by bvahedy** [comment]
I don't know much about this language, but it could be somewhere in the
code where every time a page up/down happens the code goes through a
subroutine to check for everything that it already has and therefore it
slows down what goes on the page.

**Comment by tuomas2** [comment]
I believe this could still appear, unless WebView / Chrome has been fixed.

**Comment by timbze** [comment]
Outdated
